diff --git a/node_modules/knex/lib/client.js b/node_modules/knex/lib/client.js
index 1d076bd..0977359 100644
--- a/node_modules/knex/lib/client.js
+++ b/node_modules/knex/lib/client.js
@@ -239,7 +239,7 @@ Object.assign(Client.prototype, {
     return { min: 2, max: 10, propagateCreateError: true };
   },
 
-  getPoolSettings(poolConfig) {
+  getPoolSettings(poolConfig, config) {
     poolConfig = defaults({}, poolConfig, this.poolDefaults());
 
     POOL_CONFIG_OPTIONS.forEach((option) => {
@@ -286,7 +286,7 @@ Object.assign(Client.prototype, {
     return Object.assign(poolConfig, {
       create: async () => {
         await updatePoolConnectionSettingsFromProvider();
-        const connection = await this.acquireRawConnection();
+        const connection = await this.acquireRawConnection(config);
         connection.__knexUid = uniqueId('__knexUid');
         if (poolConfig.afterCreate) {
           await promisify(poolConfig.afterCreate)(connection);
@@ -318,7 +318,7 @@ Object.assign(Client.prototype, {
     }
 
     const tarnPoolConfig = {
-      ...this.getPoolSettings(config.pool),
+      ...this.getPoolSettings(config.pool, config),
     };
     // afterCreate is an internal knex param, tarn.js does not support it
     if (tarnPoolConfig.afterCreate) {
